src/wallet/wallet.cpp

```diff
@@ -1818,32 +1818,37 @@ CWallet::ScanResult CWallet::ScanForWalletTransactions(const uint256& start_bloc
     return result;
 }
 
+std::vector<CWalletTx*> CWallet::GetSortedTxs()
```

Comment:
Should we specify what kind of sorting is applied?

Reply:
I was about to add the same suggestion 👍

Reply:
The function has ended up being removed.